lib/matplotlib/__init__.py

```diff
@@ -215,7 +217,77 @@ def _is_writable_dir(p):
     """
     return os.access(p, os.W_OK) and os.path.isdir(p)
 
+_verbose_msg = ("--verbose-LEVEL and the Verbose class are deprecated. "
+                "This functionality is now provided by the standard "
+                "python logging library.  To get more (or less) logging "
+                " output:\n"
+                "`import logging`\n "
```

Comment:
probably no backquotes, or only around the whole expression.  no dot at the end of the next line